shared/common-adapters/input.js.flow

```diff
@@ -17,6 +17,7 @@ export type Props = {
   onClick?: (event: Event) => void,
   onChangeText?: (text: string) => void,
   onEnterKeyDown?: (event: Object) => void,
+  onFocus?: () => void,
```

Comment:
Had to add this, so that clicking away from the filter input box properly calls _startEditing